boards/px4/fmu-v2/default.cmake

```diff
@@ -83,7 +83,7 @@ px4_add_board(
 		battery_status
 		sensors
 		vmount
-		vtol_att_control
+		#vtol_att_control
```

Comment:
I don't see why this is necessary, similarly with the other V2 changes. You haven't touched any code here which would increase binary sizes for these platforms.

Reply:
The v2 build was failing so I cherry-picked this commit from #13351. It is possible that I removed more than is necessary for this PR alone

Reply:
Looks like we can keep it in this PR :+1